pkg/systemd/terminal.css

```diff
@@ -13,9 +13,27 @@ html, body {
     bottom: 0px;
     left: 0px;
     right: 0px;
-    background-color: black;
 }
 
+.dark {
+    background-color: #002b36;
+}
+
+.light {
+    background-color: #fdf6e3;
+}
```

Comment:
I take it we can't put these into the themes because of CSP? Can these be prefixed with `theme-`, to avoid overlaps with other classes when our theme list grows?